packages/okta-adapter/src/change_validators/group_schema_modify_base_fields.ts

```diff
@@ -0,0 +1,46 @@
+/*
+*                      Copyright 2023 Salto Labs Ltd.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { ChangeError, ChangeValidator, InstanceElement, ModificationChange, getChangeData, isEqualValues, isInstanceChange, isModificationChange } from '@salto-io/adapter-api'
+import { collections } from '@salto-io/lowerdash'
+import { GROUP_SCHEMA_TYPE_NAME } from '../constants'
+
+const { awu } = collections.asynciterable
+
+const isBaseFieldModified = (
+  change: ModificationChange<InstanceElement>
+): boolean => {
+  const { before, after } = change.data
+  const beforeBase = before.value.definitions.base
```

Comment:
value is from type `any`, so it's not guaranteed `definitions` will exist. this way in case it doesn't the program will not crash 
```suggestion
  const beforeBase = before.value.definitions?.base
```

Reply:
I wrote the question mark here:`after.value?.definitions.base`.

Reply:
@idoamit8 I don't think it's enough, even if `value` is defined it might still be possible that `definitions` isn't
Also, if i'm not wrong, `value` is part of the type definition so it's not necessary